src/lsp.rs

```diff
@@ -20,7 +20,11 @@ pub async fn request_quote(
     p2p_address: &P2PAddress,
     refund_address: &Address,
 ) -> Result<Quote, LspError> {
-    let https = hyper_tls::HttpsConnector::new();
+    let https = hyper_rustls::HttpsConnectorBuilder::new()
+        .with_native_roots()
```

Comment:
See if this works as-is. If not, try `.with_webpki_roots()`

Reply:
I'm almost certain webpki supports letsencrypt because mozilla is a letsencrypt supporter